files/default.conf

```diff
@@ -1,5 +1,6 @@
 server {
     listen       80 default_server;
+    listen       443 default_server;
```

Comment:
You'll need the `ssl` keyword for https:

```nginx
listen 443 default_server ssl;
```

I'm not sure this will actually work, because you also need an ssl certificate. I recommend these settings:

```nginx
ssl                     on;
ssl_certificate_key     /path/to/ssl-key.pem;
ssl_certificate         /path/to/ssl-certificate-chain-without-trusted-root.pem;
ssl_trusted_certificate /path/to/ssl-certificate-chain-without-certificate-itself.pem;
```

But this role doesn't supply ssl certificates.

I am in favor of having support for this in the role, but maybe make it optional? Let's add the following default variables:

```yaml
nginx_set_default_ssl_server: no

nginx_default_ssl_certificate_key:     "/path/to/ssl-key.pem"
nginx_default_ssl_certificate:         "/path/to/ssl-certificate-chain-without-trusted-root.pem"
nginx_default_ssl_trusted_certificate: "/path/to/ssl-certificate-chain-without-certificate-itself.pem"
```

Please also update the readme to reflect these changes :slightly_smiling_face: